src/Browscap/Data/Expander.php

```diff
@@ -237,8 +270,9 @@ private function parseChildren(string $ua, array $uaDataChild, bool $lite = true
                 $uaBase = str_replace('#PLATFORM#', $platformProperties->getMatch(), $uaDataChild['match']);
 
                 if (array_key_exists('engine', $uaDataChild)) {
-                    $engine           = $this->collection->getEngine($uaDataChild['engine']);
-                    $engineProperties = $engine->getProperties();
+                    $engine                    = $this->collection->getEngine($uaDataChild['engine']);
+                    $engineProperties          = $engine->getProperties();
+                    $this->patternId['engine'] = $uaDataChild['engine'];
```

Comment:
Not necessary (imo), but if you add here, need to make sure it's in the reset array.

Reply:
fixed